nifi-assembly/pom.xml

```diff
@@ -637,6 +637,12 @@ language governing permissions and limitations under the License. -->
             <version>1.7.0-SNAPSHOT</version>
             <type>nar</type>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-cdc-mssql-nar</artifactId>
+            <version>1.7.0-SNAPSHOT</version>
```

Comment:
Sorry this has taken so long, can you rebase against the latest master and update the version(s) to 1.8.0-SNAPSHOT? Please and thanks!

Reply:
Rebased and version numbers updated. I had some files that I had failed to update from 1.6.0 to 1.7.0, so it wasn't even building correctly as it was... Builds and seems OK now.